src/Liquidator.sol

```diff
@@ -276,31 +277,38 @@ contract Liquidator is Owned, ReentrancyGuard, ILiquidator {
      * @param account The contract address of the Account being liquidated.
      * @param askedAssetAmounts Array with the assets-amounts the bidder wants to buy.
      * @param endAuction_ Bool indicating that the auction can be ended after the bid.
+     * @param data Data passed through back to the bidder via the bidCallback() call.
      * @dev We use a Dutch auction: price of the assets constantly decreases.
      * @dev The "askedAssetAmounts" array should have equal length as the stored "assetAmounts" array.
      * An amount 0 should be passed for assets the bidder does not want to buy.
      * @dev The bidder is not obliged to set endAuction to True if the account is healthy after the bid,
      * but they are incentivised to do so by earning an additional "auctionTerminationReward".
+     * @dev If the bidder is a contract, it must implement the IBidCallback interface.
```

Comment:
sync function & comments with fix/issue-161

Reply:
or merge later indeed

Reply:
This PR can now be closed actually, since we will merge it via fix/issue-161